Sources/Tests/EndToEnd/Test_EndToEnd.c

```diff
@@ -72,7 +72,6 @@ static void ReadTemplate(const char *expectedFileName, Template *expectedTemplat
     assert(ret != EOF);
 }
 
-
```

Comment:
Hi @fffej , the rule is pretty simple, if you allocate under unity then free under unity, if you malloc normally then free normally.
As @fatboab says it seems like Unity does it's own tracking, so you cann't free under Unity if it wasn't allocated under Unity. I am sure that there is ways of disabling this feature or turning it off or enhancing Unity, but I did not look at that...